### ISSUE TITLE
Look for config file in home directory

### ISSUE THREAD

**Issue by danielbrauer** [JonasJurczok/faketorio#77]
I think the standard place for a .faketorio file should be in the user's home directory, since it contains information which is not specific to any mod. It would be useful if faketorio looked there by default, so I didn't have to pass the path in with every invocation.

**Comment by JonasJurczok** [comment]
Yeah, I initially wanted to do it this way but it turned out to be surprisingly complicated to consistently find the home directory on different operating systems.
But yes, it's definitely something I want to do (or would like to review pull requests for ;) ).

**Comment by danielbrauer** [comment]
I'll happily try to add this. How do I run faketorio from a local repo?

**Comment by JonasJurczok** [comment]
Awesome :)

How I do it is I create a rockspec file from the template using `sed "s/VERSION/<new version>/g" rockspec.template > faketorio-<new version>-1.rockspec`.
Then you can run a `luarocks make <rockspec file>` and it will be installed locally. Then you can call it as if you would have installed it from luarocks itself.

Afterwards just do a `luarocks remove faketorio && luarocks install faketorio` to get back to the latest stable version.

Have fun tinkering :)

**Comment by danielbrauer** [comment]
Ah, `make` was the command I was missing. That's nice and easy, thanks!
